### ISSUE TITLE
Katalogus error getting settings for dns-records boefje

### ISSUE THREAD

**Issue by dekkers** [minvws/nl-kat-coordination#3224]
**Describe the bug**

**To Reproduce**
Go to the dns-records boefje detail page or try to run boefje.

**OpenKAT version**
This was on a KAT install that was running main before #3118 was merged and upgraded to current main.

**Screenshots**
```
katalogus-1  | [2024-07-10 13:33:27 +0000] [9] [INFO] [h11_impl] 172.30.0.6:45102 - "GET /v1/organisations/test/dns-records/settings HTTP/1.1" 500
katalogus-1  | [2024-07-10 13:33:27 +0000] [9] [ERROR] [h11_impl] Exception in ASGI application
katalogus-1  | Traceback (most recent call last):
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/uvicorn/protocols/http/h11_impl.py", line 407, in run_asgi
katalogus-1  |     result = await app(  # type: ignore[func-returns-value]
katalogus-1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/uvicorn/middleware/proxy_headers.py", line 69, in __call__
katalogus-1  |     return await self.app(scope, receive, send)
katalogus-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/fastapi/applications.py", line 1054, in __call__
katalogus-1  |     await super().__call__(scope, receive, send)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/applications.py", line 123, in __call__
katalogus-1  |     await self.middleware_stack(scope, receive, send)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/errors.py", line 186, in __call__
katalogus-1  |     raise exc
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/errors.py", line 164, in __call__
katalogus-1  |     await self.app(scope, receive, _send)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/exceptions.py", line 65, in __call__
katalogus-1  |     await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/_exception_handler.py", line 64, in wrapped_app
katalogus-1  |     raise exc
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
katalogus-1  |     await app(scope, receive, sender)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 756, in __call__
katalogus-1  |     await self.middleware_stack(scope, receive, send)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 776, in app
katalogus-1  |     await route.handle(scope, receive, send)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 297, in handle
katalogus-1  |     await self.app(scope, receive, send)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 77, in app
katalogus-1  |     await wrap_app_handling_exceptions(app, request)(scope, receive, send)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/_exception_handler.py", line 64, in wrapped_app
katalogus-1  |     raise exc
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
katalogus-1  |     await app(scope, receive, sender)
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 72, in app
katalogus-1  |     response = await func(request)
katalogus-1  |                ^^^^^^^^^^^^^^^^^^^
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/fastapi/routing.py", line 296, in app
katalogus-1  |     content = await serialize_response(
katalogus-1  |               ^^^^^^^^^^^^^^^^^^^^^^^^^
katalogus-1  |   File "/usr/local/lib/python3.11/site-packages/fastapi/routing.py", line 155, in serialize_response
katalogus-1  |     raise ResponseValidationError(
katalogus-1  | fastapi.exceptions.ResponseValidationError: 1 validation errors:
katalogus-1  |   {'type': 'dict_type', 'loc': ('response',), 'msg': 'Input should be a valid dictionary', 'input': True}

```

**Comment by underdarknl** [comment]
Would this be fixed by:
https://github.com/minvws/nl-kat-coordination/pull/3227

**Comment by dekkers** [comment]
> Would this be fixed by: #3227

Yes, it might be caused by the migration that was incomplete because it crashed. It was my dev environment which has been reset so I can't easily verify. I think that if we don't see this error when testing migrations after merging that PR we should just close this issue.

**Comment by underdarknl** [comment]
We have not seen any new reports around this. I'm closing the issue.
